### ISSUE TITLE
Spacing Incorrect

### ISSUE THREAD

**Issue by embersrise99** [AF-VCD/pdf-bullets#54]
Bullets are auto spacing incorrectly. 

Example pulled from output:

- Expos'd eqpmt deficiencies; sourced $2K in eqpmt/ tools/restor'd $204M kt mx capes--anchor'd UTAP-22 dplymt/zero delays **_(too long for 1206/EPR)*_* but its shows that it will fit in bullet writer output.

Hopefully you can resolve this, I really like your work.

**Comment by Teh-Bobo** [comment]
By chance did you try this on a different site than the [github page](https://af-vcd.github.io/pdf-bullets/)? I saw my flight chief using a different website that seems to have cloned this repo, but is not updating it and is currently hosting an older version. 

I ask because I put that bullet into the current page and it shows the correct output (that it won't fit).

**Comment by embersrise99** [comment]
No, I'm using it on the _original_ https://af-vcd.github.io/pdf-bullets/ page. I'm using it at work (govt) comp, however this worked flawlessly last year.

**Comment by Teh-Bobo** [comment]
Try refreshing the cached version of the page (ctrl + F5). If that doesn't work, which browser are you using? I can't seem to find one that doesn't work for me, but when I tried Chrome here, it wouldn't load until I did a cache refresh.

**Comment by embersrise99** [comment]
I tried using Internet Explorer, Chrome(cached) Fire Fox, and Edge. They all did the same thing. It's crazy, it has both text boxes too big and the autospace makes it extend instead of fit. When the bullet turns black after writing the length of the box, it's out put is way to long. When I get to the office tomorrow I'll screen shot it and send, so you can see the correct bullet size and the size of the box.

**Comment by ckhordiasma** [comment]
Hi, could you send me the PDF of the 1206/EPR/OPR that you are trying to put this bullet into? You should be able to upload it into a comment on this issue, or email it to me at ckhordiasma@gmail.com

**Comment by ckhordiasma** [comment]
Also, can you go to save -> as JSON and also upload that as well?

**Comment by embersrise99** [comment]
So, I have noticed it works on other co-workers computers. The spacing is normal and it works as advertised. However my chrome, edge, etc. will not work correctly with it (spacing issue)

**Comment by ckhordiasma** [comment]
got user feedback from another bullet that does not fit correctly 

- Managed reoccurring tm trng; assigned 7 topics/sparked critical thinking discussions--sharpened knowledge baseline f/14 mbrs
